2021-offseason/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+  /** Creates a new ExampleSubsystem. */
+
+  private TalonSRX intake;
```

Comment:
nit: naming: let's name this something more descriptive, like `intakeTalon` or something to that effect. since the class name is already `Intake`, it's kinda redundant to just have a member variable named `intake`